friends/src/App.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import './App.css';
+import axios from 'axios';
+import Friend from './components/friends'
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      friends: [],
+      name: '',
+      age: '',
+      email: '',
+      id: ''
+    }
+  }
+
+  componentDidMount() {
+    axios
+      .get('http://localhost:5000/friends')
+      .then(response => {
+        console.log(response)
+        this.setState({ friends: response.data })
+      })
+  }
+  onChangeHandler = e => {
+    const name = e.target.name;
```

Comment:
I like that you're being explicit here, but if you are going to break `e.target.name` into a variable I might name it something more descriptive like `inputName`, only b/c the word `name` is all over this app, and it could make debugging a pain later on. The line below is equivalent to line #28 
`this.setState({ [e.target.name]: e.target.value })`